pkg/persistence/account/load.go

```diff
@@ -0,0 +1,100 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package account
+
+import (
+	"fmt"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/files"
+	"github.com/mitchellh/mapstructure"
+	"github.com/spf13/afero"
+	"gopkg.in/yaml.v2"
+	"io"
+)
+
+// Load loads account management resources from YAML configuration files
+// located within the specified root directory path. It parses the YAML files, extracts policies,
+// groups, and users data, and organizes them into a AMResources struct, which is then returned.
+func Load(fs afero.Fs, rootPath string) (*AMResources, error) {
```

Comment:
I'd mark this as excluded from the gocognit check. 
I've quickly tried if it would actually become nicer to read if we split out the parts into `loadPolicies`, `loadGroups`, .. methods. But with error handling still needed there's not much of a difference at all, actually it becomes messier to read.

Reply:
yes the "complexity" is absolutely manageable